src/plugins/intel_cpu/src/nodes/strided_slice.cpp

```diff
@@ -310,15 +310,19 @@ void StridedSlice::prepareParams() {
             dstMemory.push_back(getChildEdgeAt(i)->getMemoryPtr());
         }
     }
-
-    execPtr = std::make_shared<StridedSliceCommonExecutor>(attrs, srcMemory, dstMemory, errorPrefix);
+    if (srcMemory[DATA_ID]->GetShape().hasZeroDims() || dstMemory[0]->GetShape().hasZeroDims())
```

Comment:
We do have isExecutable() method in Node API which was designed for such use-cases. Can we deal with the issue by its implementaion adjustment?

Reply:
Good idea, I will try it, thx!